website/content/api-docs/acl/auth-methods.mdx

```diff
@@ -71,11 +77,9 @@ The corresponding CLI command is [`consul acl auth-method create`](/commands/acl
   For more information on configuring specific auth method types, see the [auth
   method documentation](/docs/security/acl/auth-methods).
 
-- `Namespace` `(string: "")` <EnterpriseAlert inline /> - Specifies the namespace to
-  create the auth method within. If not provided in the JSON body, the value of
-  the `ns` URL query parameter or in the `X-Consul-Namespace` header will be used.
-  If not provided, the namespace will be inherited from the request's ACL
-  token or will default to the `default` namespace. Added in Consul 1.7.0.
+- `Namespace` `(string: "")` <EnterpriseAlert inline /> - Specifies the namespace of
+  the auth method you create. This takes precedence over the `ns` query parameter,
```

Comment:
Changing to "This field" (instead of "This parameter") to be consistent with copy for other entries in the "JSON Request Body Schema" section that refer to themselves as "fields".